### PR TITLE
Inct 153

### PR DESCRIPTION
**Pull request by nargiz92**

### CODE REVIEW

File: src/app/services/app-slice.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      state.message = ''
+      state.variant = 'info'
+    },
+    setAlert: (state, action: PayloadAction<{ message: string; variant: 'error' | 'info' }>) => {
+      state.message = action.payload.message
+      state.variant = action.payload.variant
+    },
+  },
+})
+export const { clearAlert, setAlert } = appSlice.actions
```
Comment (elem15) [inline src/app/services/app-slice.ts:26]: export const selectAlert = (state: RootState) => state.appSlice

File: src/widgets/alertContainer/notificationContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { clearAlert } from '@/app/services'
+import { Alert } from '@/shared/components/alert/Alert'
+import { useAppDispatch, useAppSelector } from '@/shared/lib'
+
+export const NotificationContainer = () => {
+  const message = useAppSelector(state => state.appSlice.message)
+  const variant = useAppSelector(state => state.appSlice.variant)
```
Comment (elem15) [inline src/widgets/alertContainer/notificationContainer.tsx:7]: const { message, variant } = useAppSelector(selectAlert)

File: src/shared/components/alert/Alert.module.scss [hunk 3]
```diff
@@ -1,15 +1,14 @@
 .box {
   position: fixed;
-  z-index: 100;
+  z-index: 200;
   inset: 0;
-  background-color: rgb(0 0 0 / 30%);
```
Comment (elem15) [inline src/shared/components/alert/Alert.module.scss:5]: User can't click any elements except close button. So background opacity is preferred

File: src/shared/components/alert/Alert.tsx [hunk 4]
```diff
@@ -1,39 +1,40 @@
...
 } & ComponentProps<'div'>
-export const Alert: FC<Props> = ({ children, variant, ...rest }) => {
-  const [open, setOpen] = useState(true)
-
-  if (open) {
-    return (
+export const Alert: FC<Props> = ({ message, onClose, variant = 'error', ...rest }) => {
+  if (message) {
+    return createPortal(
       <div className={s.box}>
```
Comment (elem15) [inline src/shared/components/alert/Alert.tsx:18]: close alert by clicking anywhere  onClick={onClose} ???

Comment (nargiz92) [reply src/shared/components/alert/Alert.tsx:18]: No

File: src/shared/components/alert/Alert.module.scss (hunk 3 above)
Comment (nargiz92) [reply src/shared/components/alert/Alert.module.scss:5]: I fix it

**Review by elem15 — APPROVED** [review]

**Pull request merged**